Ginger/Ginger/ALM/ALMConnectionPage.xaml.cs

```diff
@@ -166,6 +175,13 @@ public void RefreshALMSolutionSettings()
                 UserNameTextBox.Text = "";
                 StyleRadioButtons();
             }
+            else if ((ALMIntegration.eALMType)WorkSpace.Instance.Solution.AlmType == ALMIntegration.eALMType.Qtest && qTestRadioButton.IsChecked == false)
```

Comment:
@Arvind- you are clearing a binding on ALM type change. so please check if this function is still required or can be removed